cohorts/file_helpers.py

```diff
@@ -195,6 +173,13 @@ def cohort_files(cohort_id, inc_filters=None, user=None, limit=25, page=1, offse
                         })
             filter_counts = counts
         else:
+            case_barcode = None
+            case_barcode_condition = ''
+            if 'case_barcode' in inc_filters:
+                case_barcode = ''.join(inc_filters['case_barcode'])
+                del inc_filters['case_barcode']
+                case_barcode_condition = " AND LOWER(cs.case_barcode) LIKE LOWER(%s)"
+
```

Comment:
Quick question--is the case barcode being sent from the request with the % now?

Reply:
See view.py line 1629 where it is formatted and added to inc_filters['case_barcode']:


        if panel_type:
            inc_filters = json.loads(request.GET.get('filters', '{}')) if request.GET else json.loads(
                request.POST.get('filters', '{}'))
            if request.GET.get('case_barcode', None):
                inc_filters['case_barcode'] = ["%{}%".format(request.GET.get('case_barcode')),]

------